### PR TITLE
[Rework] split ParameterLine

### PR DESCRIPTION
**Pull request by capyq**

### CODE REVIEW

**Comment by jonenst** [comment]
rename file to parameter-group instead of Parameter-group ?

**Comment by capyq** [comment]
> rename file to parameter-group instead of Parameter-group ?

done

**Review by jonenst — APPROVED** [review]

File: src/components/dialogs/parameters/map-parameters.jsx [hunk 1]
```diff
@@ -4,25 +4,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-
+//TODO: File in JSX because @powsybl/diagram-viewer doesn't provide type. change to TSX when types are awailable.
```
Comment (Tristan-WorkGH) [inline src/components/dialogs/parameters/map-parameters.jsx:7]: Can be done with v0.5.4 of diagram-viewer merged in #2177

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]

File: src/components/dialogs/parameters/map-parameters.jsx (hunk 1 above)
Comment (capyq) [reply src/components/dialogs/parameters/map-parameters.jsx]: done

**Review by Tristan-WorkGH — APPROVED** [review]

File: src/components/dialogs/parameters/voltageinit/equipment-selection-parameters.tsx [hunk 2]
```diff
@@ -5,46 +5,37 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { ElementType } from '@gridsuite/commons-ui';
+import { ElementType, EquipmentType } from '@gridsuite/commons-ui';
```
Comment (sBouzols) [inline src/components/dialogs/parameters/voltageinit/equipment-selection-parameters.tsx:8]: Ok but it's not exactly the same enum, there is no TIE_LINE in the commons-ui enum.

**Pull request merged**